src/Symfony/Component/Translation/Command/XliffLintCommand.php

```diff
@@ -124,7 +124,9 @@ private function validate($content, $file = null)
             $normalizedLocale = preg_quote(str_replace('-', '_', $targetLanguage), '/');
             // strict file names require translation files to be named '____.locale.xlf'
             // otherwise, both '____.locale.xlf' and 'locale.____.xlf' are allowed
-            $expectedFilenamePattern = $this->requireStrictFileNames ? sprintf('/^.*\.%s\.xlf/', $normalizedLocale) : sprintf('/^(.*\.%s\.xlf|%s\..*\.xlf)/', $normalizedLocale, $normalizedLocale);
+            // also, the regexp matching must be case-insensitive, as defined for 'target-language' values
+            // http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#target-language
+            $expectedFilenamePattern = $this->requireStrictFileNames ? sprintf('/^.*\.%s\.xlf/i', $normalizedLocale) : sprintf('/^(.*\.%s\.xlf|%s\..*\.xlf)/i', $normalizedLocale, $normalizedLocale);
```

Comment:
Do we want the `.xlf` extension to be case-insensitive as well?

What about something like this instead? `/^.*\.(?i:%s)\.xlf/`

Reply:
Good catch! I haven't thought about that. Changed. Thanks.